.env

```diff
@@ -0,0 +1,2 @@
+FIRMWARE_VERSION=2021.11.22.0-1
+DIAGNOSTICS_VERSION=b30bd8a
```

Comment:
We discussed adding a Nebra version. Not sure what format it will take though

Reply:
https://github.com/NebraLtd/hm-miner/issues/47

Reply:
Probably not needed anymore as incrementing the main firmware version seems a better option at least for now